src/Compilers/CSharp/Portable/Emitter/Model/ArrayTypeSymbolAdapter.cs

```diff
@@ -31,7 +31,7 @@ bool Cci.IArrayTypeReference.IsVector
         {
             get
             {
```

Comment:
Do we own `IArrayTypeReference`?  If so should we rename `IsVector` to `IsSZArray` for consistency?